modules/statics/src/account.ts

```diff
@@ -592,3 +618,75 @@ export function hederaCoin(
     })
   );
 }
+
+/**
+ * Factory function for ALGO token instances.
+ *
+ * @param name unique identifier of the token
+ * @param fullName Complete human-readable name of the token
+ * @param decimalPlaces Number of decimal places this token supports (divisibility exponent)
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a coin.
+ * @param assetURL Optional asset Url for more informationa about the asset
+ * See https://developer.algorand.org/docs/reference/transactions/#url
+ * @param prefix? Optional token prefix. Defaults to empty string
+ * @param suffix? Optional token suffix. Defaults to token name.
+ * @param network? Optional token network. Defaults to ALGO mainnet.
+ * @param features? Features of this coin. Defaults to the DEFAULT_FEATURES defined in `AccountCoin`
+ * @param primaryKeyCurve The elliptic curve for this chain/token
+ */
+export function algoToken(
+  name: string,
+  fullName: string,
+  decimalPlaces: number,
+  asset: UnderlyingAsset,
+  assetURL: string = '',
+  features: CoinFeature[] = AccountCoin.DEFAULT_FEATURES,
+  prefix: string = '',
+  suffix: string = name.toUpperCase(),
+  network: AccountNetwork = Networks.main.algorand,
+  primaryKeyCurve: KeyCurve = KeyCurve.Ed25519
+) {
+  return Object.freeze(
+    new AlgoCoin({
+      name,
+      fullName,
+      decimalPlaces,
+      asset,
+      assetURL,
+      features,
+      prefix,
+      suffix,
+      network,
+      isToken: true,
+      primaryKeyCurve,
+    })
+  );
+}
+
+/**
+ * Factory function for testnet ALGO token instances.
+ *
+ * @param name unique identifier of the token
+ * @param fullName Complete human-readable name of the token
+ * @param decimalPlaces Number of decimal places this token supports (divisibility exponent)
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a coin.
+ * @param assetURL Optional asset Url for more informationa about the asset
+ * See https://developer.algorand.org/docs/reference/transactions/#url
+ * @param prefix? Optional token prefix. Defaults to empty string
+ * @param suffix? Optional token suffix. Defaults to token name.
+ * @param network? Optional token network. Defaults to ALgo testnet.
+ * @param features? Features of this coin. Defaults to the DEFAULT_FEATURES defined in `AccountCoin`
+ */
+export function talgoToken(
+  name: string,
+  fullName: string,
+  decimalPlaces: number,
+  asset: UnderlyingAsset,
+  assetURL: string = '',
+  features: CoinFeature[] = AccountCoin.DEFAULT_FEATURES,
+  prefix: string = '',
+  suffix: string = name.toUpperCase(),
+  network: AccountNetwork = Networks.test.algorand
+) {
+  return stellarToken(name, fullName, decimalPlaces, asset, assetURL, features, prefix, suffix, network);
```

Comment:
would be good to document why we use stellarToken here for newcomers that aren't familar.

Reply:
Have updated it ti be talgoToken - had missed to add it after my change in earlier push.